packages/kmod-6.12-amdgpu/kmod-6.12-amdgpu.spec

```diff
@@ -0,0 +1,72 @@
+%global kernel_major 6.12
+%global kernel_sources %{_cross_usrsrc}/kernels/%{kernel_major}
+%global _cross_kver %{kernel_major}.55
+%global _cross_kmoddir %{_cross_libdir}/modules/%{_cross_kver}
+%global _ko ko
+
+Name: %{_cross_os}kmod-6.12-amdgpu
+Version: 30.10
+Release: 1%{?dist}
+Summary: AMD GPU drivers for the 6.12 kernel
+License: GPL-2.0 WITH Linux-syscall-note
+URL: https://www.amd.com/
+
+Source0: https://repo.radeon.com/amdgpu/30.10.2/el/10/main/x86_64/amdgpu-dkms-6.14.14-2226257.el10.noarch.rpm
+Source1: COPYING
+
+BuildRequires: %{_cross_os}kernel-6.12-devel
+Requires: %{_cross_os}kernel-6.12
+Requires: %{_cross_os}linux-firmware-amd
+
+%description
+%{summary}.
+
+%prep
+# Extract DKMS source from RPM
+rpm2cpio %{S:0} | cpio -idmu './usr/src/amdgpu-*'
+find usr/src/ -mindepth 1 -maxdepth 1 -type d -exec mv {} amdgpu \;
+rm -r usr
+
+# Apply Bottlerocket DKMS configuration
+pushd amdgpu
+
+# Configure DKMS
+KERNELVER=%{_cross_kver} amd/dkms/configure --with-linux=%{kernel_sources}
```

Comment:
You could use the `cross_configure` macro and likely avoid the flags in the `make modules` call.